app/views/pair_requests/create.turbo_stream.erb

```diff
@@ -0,0 +1,32 @@
+<%= turbo_stream.update 'new_request_form' do %>
+  <%= render 'form', pair_request: @pair_request %>
+<% end %>
+
+
+<% if @new_pair_request %>
```

Comment:
We shouldn't need this check?

Reply:
I wasn't sure about that, but figured better safe than sorry. I'll try triggering an error and see whether the request/response cycle complains. If not, better to remove.

Reply:
After checking, it seems important to keep here. I think we'll be able to remove once we change the error to be a flash stream.

Reply:
#102 should address this issue